node-problem-detector-0.8.yaml

```
@@ -1,13 +1,15 @@
 package:
   name: node-problem-detector-0.8
   version: 0.8.20
-  epoch: 8
+  epoch: 9
   description: node-problem-detector aims to make various node problems visible to the upstream layers in the cluster management stack.
   copyright:
     - license: Apache-2.0
   dependencies:
     runtime:
       - busybox
+      - health-checker-${{vars.major-minor-version}}=${{package.full-version}}
+      - log-counter-${{vars.major-minor-version}}=${{package.full-version}}
```

we can omit package.full-version while specifying runtime deps. 
```suggestion
      - health-checker-${{vars.major-minor-version}}
      - log-counter-${{vars.major-minor-version}}
```

---

thank you for the review, omitted the package.full-version, PTAL